packages/shared/src/hooks/useSquadChecklist.tsx

```diff
@@ -69,16 +72,21 @@ const useSquadChecklist = ({
         },
         actions,
       }),
-      [ActionType.SquadFirstPost]: createChecklistStep({
-        type: ActionType.SquadFirstPost,
-        step: {
-          title: 'Share your first post',
-          description:
-            'Share your first post to help other squad members discover content you found interesting.',
-          component: SharePostChecklistStep,
-        },
-        actions,
-      }),
+      [ActionType.SquadFirstPost]:
+        verifyPermission(squad, SourcePermissions.Post) &&
```

Comment:
Could you check with Sab on this please.

What happens when squad doesn't allow posting, but user is new, should we simply render different options? (or not maybe it's to edge-case)